README.md

```diff
@@ -29,15 +29,15 @@ Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps gi
 
 3.  Create a symlink of the repository in the global folder
```

Comment:
"Add a symbolic link to the global folder"

Re-commenting this so that you don't miss it again :)